conf/test_full.config

```diff
@@ -12,8 +12,6 @@ params {
   config_profile_description = 'Full test COSMIC generation'
 
   // Input data for full size test
```

Comment:
Maybe the other functions of the pipeline would need a test case as well for example the vcf input or the various options that can be selected.

Reply:
We have tried to add more tests but the size in Github for most of those databases is huge.